erpnext/patches/v7_1/fix_start_end_date_salary_register.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import unicode_literals
+import frappe
+from frappe import _
+from calendar import monthrange
+
+def execute():
+	sql = """select employee, month, fiscal_year, status, employee_name, name
+		from `tabSalary Slip`
+		where start_date is null
+		and end_date is null"""
+
+	for date in frappe.db.sql(sql, as_dict=True):
+		years = date.fiscal_year.split("-")
+		if int(date.month) >= 7:
```

Comment:
we need start_date and end_date based on fiscal_year
standard has changed monthly salary register report to salary register
it was removed month filter and added a datepicker
so it will be a temporary fix